packages/dds/tree/src/test/util/summaryBenchmarkSizes.ts

```diff
@@ -0,0 +1,142 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { makeRandom } from "@fluid-internal/stochastic-test-utils";
+import { IsoBuffer } from "@fluidframework/common-utils";
+import { TransactionResult } from "../../checkout";
+import { singleTextCursor } from "../../feature-libraries";
+import { ISharedTree } from "../../shared-tree";
+import { detachedFieldAsKey, TreeValue } from "../../tree";
+import { brand } from "../../util";
+import { TestTreeProvider } from "../utils";
+
+const random = makeRandom(129384);
```

Comment:
is there a reason why this number was chosen? When picking an arbitrary number and we don't care what it is as long as its consistent, might be nice for the API to have a default, or to simple pass in 0.

Reply:
Set seed to 0